packages/contracts/src/test/plugin/UUPSUpgradeable/PluginUUPSUpgradeableSetupMock.sol

```diff
@@ -42,10 +42,11 @@ contract PluginUUPSUpgradeableSetupV1Mock is PluginSetup {
 }
 
 contract PluginUUPSUpgradeableSetupV1MockBad is PluginUUPSUpgradeableSetupV1Mock {
+    
     function prepareInstallation(
         address _dao,
         bytes memory
-    ) public pure override returns (address plugin, PreparedSetupData memory preparedSetupData) {
+    ) public view override returns (address plugin, PreparedSetupData memory preparedSetupData) {
```

Comment:
This adds a compiler warning.

```
Warning: Unused function parameter. Remove or comment out the variable name to silence this warning.
  --> src/test/plugin/Clonable/PluginCloneableMock.sol:21:25:
   |
21 |     function initialize(IDAO _dao) external {
   |                         ^^^^^^^^^


Warning: Function state mutability can be restricted to pure
  --> src/test/plugin/UUPSUpgradeable/PluginUUPSUpgradeableSetupMock.sol:46:5:
   |
46 |     function prepareInstallation(
   |     ^ (Relevant source part starts here and spans across multiple lines).
```
Please revert and fix the other compiler warning as well.

Reply:
https://github.com/aragon/core/pull/242/commits/48b211bbbdf5d20660a929aa53fb37e89ef5c3e9 (resolve if you agree)